core/src/main/java/com/linecorp/armeria/internal/common/metric/RequestMetricSupport.java

```diff
@@ -411,6 +436,14 @@ public DistributionSummary failureAttempts() {
                                                             idPrefix.tags("result", "failure"),
                                                             distributionStatisticConfig());
         }
+
+        @Override
+        public Counter actualRequestsCause(@Nullable Throwable cause, HttpStatus status) {
+            final String causeStr = cause != null ? cause.getClass().getSimpleName() : "null";
+            return parent.counter(
+                    idPrefix.name("actual.requests.cause"),
+                    idPrefix.tags("cause", causeStr, "http.status", status.codeAsText()));
```

Comment:
note: I think we can potentially add a `direction` tag later to denote if this is `requestCause` or `responseCause`.